RZAssemblage/RZAssemblage.h

```diff
@@ -1,19 +1,127 @@
 //
-//  RZAssemblage.h
+//  RZBaseAssemblage.h
 //  RZAssemblage
 //
 //  Created by Brian King on 1/27/15.
 //  Copyright (c) 2015 Raizlabs. All rights reserved.
 //
 
-#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
 
-//! Project version number for RZAssemblage.
 FOUNDATION_EXPORT double RZAssemblageVersionNumber;
-
-//! Project version string for RZAssemblage.
 FOUNDATION_EXPORT const unsigned char RZAssemblageVersionString[];
```

Comment:
If these auto-generated version variables aren't actually defined, they should be removed.